rest-api/spring-rest/src/test/java/uk/gov/gchq/gaffer/rest/controller/OperationControllerTest.java

```diff
@@ -49,20 +49,15 @@
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 import java.util.stream.Collectors;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
-import static uk.gov.gchq.gaffer.core.exception.Status.BAD_REQUEST;
-import static uk.gov.gchq.gaffer.core.exception.Status.INTERNAL_SERVER_ERROR;
-import static uk.gov.gchq.gaffer.core.exception.Status.NOT_FOUND;
+import static uk.gov.gchq.gaffer.core.exception.Status.*;
```

Comment:
I expect your IDE did this automatically. Checkstyle doesn't allow this type of import.

Reply:
yes, my IDE did that. The "Optimize imports" option was ticked.